src/pages/recipes/[recipeId].tsx

```diff
@@ -1,14 +1,147 @@
 import { Loader } from "@/shared/components/Loader";
 import { trpc } from "@/utils/trpc";
 import { useRouter } from "next/router";
+import Image from "next/image";
+import { RiShareForwardFill } from "react-icons/ri";
+import { SnackbarProvider } from "@/components/Snackbar";
 
 export default function Recipe() {
   const router = useRouter();
   const { data, isError, isLoading } = trpc.recipes.getRecipe.useQuery({
     recipeId: String(router.query.recipeId),
   });
+
   if (isLoading) {
     return <Loader />;
   }
-  return <div>Unfinished</div>;
+
+  return (
+    <div className="flex flex-col items-center pb-10">
+      <div className="w-2/3 space-y-6">
+        <div className="text-center text-4xl font-bold">{data!.name}</div>
+        <div>
+          {data!.description}
+          <br />
+          <br />
+          <div className="text-sm">
+            Recipe by <b>{data!.author.name}</b>
+          </div>
+        </div>
+        <button
+          onClick={() => copy()}
+          className="flex flex-row border-4 border-solid border-red-400 p-2 px-10 font-bold hover:bg-red-400 gap-3"
+        >
+          Share
+          <RiShareForwardFill size={22}/>
+        </button>
+        <div>
```

Comment:
Maybe set some sort of max width and height on the div and the get Image to fill up 100% of that width. Currently, some images are really big.
![1](https://user-images.githubusercontent.com/47471007/234071611-3b400ad9-99fa-4961-ad67-2910ba57e1a3.png)